### DISCUSSION TITLE
本地修改base路径

### DISCUSSION THREAD

**Discussion by pangxzi** [umijs/umi#8673] [Q&A]
本地配置了这个以后  base: "/platformweb",
这个文件怎么也可以加这个路径
![image](https://user-images.githubusercontent.com/52907799/180362680-71e7805e-edb0-4635-95ca-6409eae54653.png)

**Comment by JesuisTong** [comment]
`umirc` 里配置 `publicPath:  "/platformweb"`

这样应该就行
